tests/cupyx_tests/scipy_tests/fftpack_tests/test_fftpack.py

```diff
@@ -108,16 +111,19 @@ def test_fft2_plan(self, xp, scp, dtype):
                                  contiguous_check=False, scipy_name='scp')
     def test_fft2_overwrite_plan(self, xp, scp, dtype):
         x = testing.shaped_random(self.shape, xp, dtype)
+        # hack: avoid testing the cases in which getting a cuFFT plan is impossible
+        if (len(self.shape) > 3) and (self.axes is None):
```

Comment:
I would just reuse `_default_plan_type` instead here as it also accounts for other cases where 'nd' may not work:

```python
from cupy.fft.fft import _default_plan_type
```

```python
if _default_plan_type(x, s=self.shape, axes=self.axes) != 'nd':
   return x
```

Reply:
Sounds good, thanks! If @asi1024 is OK with it I'll make this change.

Reply:
btw it should be `s=self.s` not `s=self.shape`.

Reply:
done